test/object-store/mongodb/functions/sumFunc/source.js

```diff
@@ -1,3 +1,3 @@
 exports = function(...args) {
-    return parseInt(args.reduce((a,b) => a + b, 0));
+    return args.reduce((a,b) => a + b, 0);
```

Comment:
The call to parseInt is not necessary after https://github.com/10gen/baas/pull/3962